docs/tools/vdb_table/data/clickhouse.json

```diff
@@ -113,9 +113,9 @@
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
     "source_url": "https://github.com/chdb-io/chdb",
-    "comment": "possible via https://github.com/chdb-io/chdb"
+    "comment": "chDB is the embedded version of ClickHouse"
```

Comment:
```suggestion
    "comment": "chDB is the embedded version of ClickHouse that's maintained by third-parties"
```

Reply:
cc @svonava-superlinked

Reply:
@prrao87 sure, makes sense to clarify - we could go with something like:

chDB is an embedded version of ClickHouse. It is a separate, independent project - not maintained by ClickHouse.